daffodil-cli/src/it/scala/org/apache/daffodil/saving/TestCLISaveParser.scala

```diff
@@ -17,253 +17,143 @@
 
 package org.apache.daffodil.saving
 
-import org.junit.Assert._
 import org.junit.Test
-import org.junit.Before
-import org.junit.After
-import org.apache.daffodil.CLI.Util
-import java.io.File
-import net.sf.expectit.matcher.Matchers.contains
-import net.sf.expectit.matcher.Matchers.eof
+
+import org.apache.daffodil.CLI.Util._
 import org.apache.daffodil.Main.ExitCode
 
 class TestCLISaveParser {
 
-  val output1 = Util.getExpectedString("output1.txt")
-  val output4 = Util.getExpectedString("output4.txt")
-  val output6 = Util.getExpectedString("output6.txt")
-  val output12 = Util.getExpectedString("output12.txt")
-  val savedParserFile = new File("savedParser.xsd.bin")
```

Comment:
This PR inlines these files' expected strings into the tests below and uses withTempFile { parser instead of savedParserFile.  Nice!